### PR TITLE
[NELP-13] Fix RTL direction issue and translations

### PR DESCRIPTION
**Pull request by shimulch**
## Description

This PR fixes content direction when using an RTL language. Also, it fixes the broken translations.

## Supporting information

https://tasks.opencraft.com/browse/NELP-13

**Screenshot**
Screenshot 2021-04-09 at 8 14 56 AM

## Testing instructions

1. Clone this PR on your local devstack - `src/` folder
2. Install the app on your LMS - `make lms-shell` then `pip install -e /edx/src/custom-student-verification`.
3. Restart LMS - `make lms-restart`
4. Change the language to `ar` from `http://localhost:18000/update_lang/`
5. Visit `http://localhost:18000/verify_student/reverify` to check if the content direction is RTL for the form.

## Deadline

ASAP

## Other information

We don't have `ar` translations for all strings used in this app.

**Reviewer**
- [ ] @giovannicimolin 
- [ ] @lgp171188

### CODE REVIEW

**Comment by shimulch** [comment]
@lgp171188, I've tested if we can simply add translations here. Yes, we can. If we add translations for `ar` in `django.po` and compile, it will load as expected. 

So if we use this PR with https://github.com/nelc/nelp-edx-theme/pull/11 it will load translations in the following order -

1. Custom theme dir
2. LMS locale dir
3. Django apps locales (ex - locale from this app)
4. Django build in locale

**Comment by giovannicimolin** [comment]
@shimulch Please add any translations here, so this can be re-used by other clients too.
Is this ready for review? If so, I can do a review pass here.

@farhaanbukhsh Can we get a translation from NELP for these files?

**Comment by shimulch** [comment]
@giovannicimolin, We don't have any translations for this. I don't think NELP did translations for this yet.

File: custom_student_verification/templates/custom_student_verification.html [hunk 1]
```diff
@@ -1,14 +1,30 @@
...
+        </form>
     {% endif %}
-    <form method="post" action="{% url 'custom_student_verification_app:custom-student-verification' %}" enctype="multipart/form-data">
-        {% csrf_token %}
-        <label for="id_id_card">ID Card:</label> {{ form.id_photo }}
-        <input type="submit" value="Submit" />
-    </form>
-{% endif %}
\ No newline at end of file
+</div>
```
Comment (giovannicimolin) [inline custom_student_verification/templates/custom_student_verification.html]: Add new line at the end of the file.

File: test_settings.py [hunk 2]
```diff
@@ -43,3 +43,9 @@ def root(*args):
 
 SECRET_KEY = 'insecure-secret-key'
 
+TEMPLATES = [
```
Comment (giovannicimolin) [inline test_settings.py:49]: This is conflicting with the current master, please rebase.

**Review by giovannicimolin — CHANGES_REQUESTED** [review]
@shimulch This is working perfectly in the devstack, but I left a few comments in the code.
Also, make sure to add a version bump on `setup.py` (minor version is ok).

Let me know when I can review again

File: custom_student_verification/templates/custom_student_verification.html (hunk 1 above)
Comment (shimulch) [reply custom_student_verification/templates/custom_student_verification.html]: Added

File: test_settings.py (hunk 2 above)
Comment (shimulch) [reply test_settings.py:49]: Rebased

**Comment by shimulch** [comment]
@giovannicimolin, I've added a version bump and addressed your review comments. This is ready to be reviewed again.

**Review by giovannicimolin — APPROVED** [review]
:+1:

- [x] I tested this:
1. In the devstack, I checked that this displays in RTL.
- [x] I read through the code
- [x] I checked for accessibility issues
- [ ] ~~Includes documentation~~ NA

@shimulch Good to go! Nice work :)
You can `Squash and merge` this now, and create a new release.

**Comment by shimulch** [comment]
@giovannicimolin, Squashed & Merged. Released 1.2.1

**Comment by farhaanbukhsh** [comment]
@giovannicimolin sorry I missed the ping, as I see @shimulch handled it. Thanks a lot, folks for all the great work. :)

**Pull request merged**